.github/workflows/main.yml

```diff
@@ -64,23 +64,20 @@ jobs:
   cypress_tests:
     name: Cypress tests
     runs-on: ubuntu-latest
-    needs: install_dependencies
+    strategy:
+      matrix:
+        browser: [chrome, firefox]
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: "14"
-      - name: Restore cache
-        uses: actions/cache@v2.1.4
-        with:
-          path: "**/node_modules"
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
       - name: Run Cypress tests
         uses: cypress-io/github-action@v2
         with:
-          start: npm run dev
+          start: npm run start
+          build: npm run build
+          headless: true
           wait-on: "http://localhost:3000"
+          browser: ${{ matrix.browser }}
```

Comment:
Iterate over browser input, so chrome and firefox.  See docs: https://github.com/cypress-io/github-action#browser